lib/puppet/functions/deprecation.rb

```diff
@@ -8,6 +8,14 @@
   end
 
   def deprecation(key, message)
+    if defined? Puppet::Pops::PuppetStack.stacktrace()
+      stacktrace = Puppet::Pops::PuppetStack.stacktrace()
+      file = stacktrace[0]
+      line = stacktrace[1]
+      output_message = "#{message} at #{file}:#{line}"
+    else
+      output_message = message
```

Comment:
Don't you need to use `output_message` in the failure msg and deprecation warning msg below?